tests/installation/product_selection/install_SLES.pm

```diff
@@ -5,11 +5,17 @@
 # Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
 
 use base 'y2_installbase';
+use testapi 'get_var';
+use YuiRestClient 'get_timeout';
```

Comment:
actually we should pass the timeout scale as parameter to not introduce dependency with YuiRestClient (sorry I missed that...) but then we would repeat that operation in several places and we need some object to encapsulate, could you please create a ticket for that?

Reply:
so, how about we move the get_timeout to the lib/YuiRestClient/App.pm, we can have set_timeout and get_timeout there, then we can use the  logic 
like: 
```
my $app = YuiRestClient::get_app(timeout => 120, interval => 1);
my $timeout = $app->get_timeout();
```

Reply:
we don't need the dependency of the App in the Test Layer neither.

Reply:
OK,  then I'd create a ticket for it. thanks

Reply:
```
$product_selection->wait_product_selection_page({timeout_scale => get_var('TIMEOUT_SCALE', 1), message => 'Product Selection page is not displayed'});
```
Then we do the multiplication in the Controller Layer, where we already have `YuiRestClient`. But this multiplication should be encapsulated in the future, you can do it like this and file the ticket just for the encapsulation ( a new object that does the operation).

Reply:
New ticket [**125561**](https://progress.opensuse.org/issues/125561) was created.